### DISCUSSION TITLE
How can I modify a Howl object while a song is playing?

### DISCUSSION THREAD

**Discussion by trevorwrightdev** [goldfire/howler.js#1585] [Q&A]
Hey, the main thing I want to accomplish is to make it so the user can determine if they want a song to loop while that song is playing. I'm storing my Howl object as a state. I am just wondering what would be the best way to accomplish this. Here is my initialization of the Howl object: 
<img width="363" alt="image" src="https://user-images.githubusercontent.com/86077703/157530249-4de6e1e2-0be4-4c8a-a9f7-7fec4272058a.png">
Here is the code I have been using to change the state of the song: 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/86077703/157529734-41569976-934c-4781-9f86-4da44f004fa4.png">
This will essentially do nothing when I test it out. I would prefer to do something like music.loop(true) but I can't seem to do that if I am using useState. How would a normal person modify the Howl object mid song?

**Comment by maruprj31** [comment]
good
